app/views/settings/preferences/show.html.haml

```diff
@@ -18,6 +18,8 @@
 
     = f.input :setting_default_sensitive, as: :boolean, wrapper: :with_label
 
+    = f.input :setting_default_license, collection: Status::LICENSE_URLS.keys, wrapper: :with_label
```

Comment:
Perhaps `Status::LICENSE_URLS.keys` is not user-friendly. Maybe this whole input is confusing. Very open to opinions here.

Reply:
It also prevents from localization. Creative Commons localizes license names on their Web sites. It is better to conform the prior instances and localize license names as they do.